build.gradle.kts

```diff
@@ -27,3 +27,55 @@ detekt {
     autoCorrect = true
     config = files("${rootDir}/detekt.yml")
 }
+
+dependencies {
+    implementation("org.jetbrains.kotlin:kotlin-stdlib")
+    implementation("com.squareup.okhttp3:okhttp:4.9.2")
+    implementation("com.google.protobuf:protobuf-java:3.17.3")
+    testImplementation("org.jetbrains.kotlin:kotlin-test")
+    testImplementation("junit:junit:4.13.2")
```

Comment:
This isn't my area of expertise, but from looking at other bits of code, I think there's a pattern of tracking dependencies via aliases to the toml.